### PR TITLE
Improvement/publishers network

### PR DESCRIPTION
**Pull request by OAuthBringer**
I've made some minor modifications to the docker-compose network configuration to allow other dockerized applications (i.e. Publishers) join the same local network as bat-ledgers and this simplify networking and allow for easier local interegration development.  There is a companion PR in Publishers that utilizes these changes so that fixture balances can easily be loaded into eyeshade with a simple makefile command.

See https://github.com/brave-intl/publishers/pull/3574 for additional context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ file when launching services.
 # Ledger listens on port 3001, eyeshade on 3002, and balance on 3003
 
 # Note you can run any subset of services (e.g. only eyeshade)
-docker-compose up eyeshade-web eyeshade-worker
+docker-compose up eyeshade-web eyeshade-consumer
 
 # You can also launch and run services in the background
-docker-compose up -d eyeshade-web eyeshade-worker
+docker-compose up -d eyeshade-web eyeshade-consumer
```
Comment (OAuthBringer) [inline README.md:27]: There is no eyeshade-worker instance defined in the docker-compose file any longer, I'm assuming this is now the eyeshade-consumer.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 version: "2.1"
 
 networks:
-  ledger:
```
Comment (OAuthBringer) [inline docker-compose.yml:4]: this is all redundant, if left empty all containers will simply by added to the default network which can then be renamed "ledger".

**Comment by OAuthBringer** [comment]
Hmm, I'm not sure what the deal with the failing builds are.  In attempting to troubleshoot I checkout master and I am getting the same timeout failures for kafka on master as with the branch

**Comment by OAuthBringer** [comment]
I can say though that these network changes allow the easy loading of fixture data from a local development context and eyeshade/publishers seem to be playing well.

**Review by husobee — APPROVED** [review]

**Pull request merged**